### PR TITLE
Bump version to 1.2.6

### PR DESCRIPTION
**Pull request by v4lux**

### CODE REVIEW

**Comment by Quetzacoalt91** [comment]
As the narketplace cannot detect properly the version in the module, shall we remove this constant from the class and write `$this->version = '1.2.6':` instead?

**Comment by Quetzacoalt91** [comment]
Well, the marketplace never took in account this kind of constant properly We're facing an improvement instead of a regression, which makes this issue not critical. And I'm not even sure they are aware of this.

In the meantime we must add some changes in the module, or we will encounter the same issue as during the previous release. :/

File: ps_checkout.php [hunk 1]
```diff
@@ -46,8 +46,11 @@
 
 class Ps_checkout extends PaymentModule
 {
+<<<<<<< Updated upstream
     const VERSION = '1.2.6';
 
+=======
+>>>>>>> Stashed changes
```
Comment (Quetzacoalt91) [inline ps_checkout.php]: Conflict marks to be removed

Comment (v4lux) [reply ps_checkout.php]: Thx, solved :)

**Pull request merged**